transform/all_collection/README.md

```diff
@@ -2,11 +2,18 @@
 Note that binary data (`.tsv`) can be transformed to JSONL format via `tsv_to_jsonl.py` [script](https://github.com/cyfronet-fid/eosc-search-service/blob/a4662788bc5f62d5f2a804727c8df517ccfbc971/transform/transform/v3/tsv_to_jsonl.py).
 ## ENV variables
 We are using .env to store user-specific constants. This file is not tracked by git and it needs to be present in the project directory. Details:
-- `DATASET_PATH` - path to datasets data file.
-- `PUBLICATION_PATH` - path to publications data file.
-- `SOFTWARE_PATH` - path to software data file.
-- `TRAININGS_PATH` - path to trainings data file.
-- `SERVICES_PATH` - path to services data file (optional).
-- `OUTPUT_PATH` - where the output should be saved? Default: `output`
+- `DATASET_DIR_PATH` - path to datasets **directory**.
+- `PUBLICATION_DIR_PATH` - path to publications **directory**.
+- `SOFTWARE_DIR_PATH` - path to software **directory**.
+- `TRAINING_FILE_PATH` - path to trainings **directory**.
+- `SERVICE_FILE_PATH` - path to services **directory**.
+- `OUTPUT_DIR_PATH` - where the output should be saved? Default: `output`.
```

Comment:
Paths can have default values.

For example:
DATASET_DIR_PATH: './${OUPUT_DIR_PATH}/datasets'

Won't be easier to have a full path instead DIR_PATH and FILE_PATH. I believe most of the use cases will be to `mv` `OUTPUT` directory content somewhere or load all files inside it.

Reply:
I can set some defaults but in my opinion, they won't be so much useful. Maybe instead of them, `env template` will be better? A file with more or less my current setup:
```
DATASET_DIR_PATH=input_data/dataset/
PUBLICATION_DIR_PATH=input_data/publication/
SOFTWARE_DIR_PATH=input_data/software/
TRAINING_FILE_PATH=input_data/trainings//
SERVICE_FILE_PATH=input_data/services/
OUTPUT_DIR_PATH=output_data
INPUT_FORMAT=json
OUTPUT_FORMAT=json
SOLR_ADDRESS=http://127.0.0.1
SOLR_PORT=8983
SOLR_DATASET_COLS="all_collection_26-09-2022 dataset_26-09-2022"
SOLR_PUBLICATION_COLS="all_collection_26-09-2022 publication_26-09-2022"
SOLR_SOFTWARE_COLS="all_collection_26-09-2022 software_26-09-2022"
SOLR_TRAINING_COLS="all_collection_26-09-2022 training_26-09-2022"
SOLR_SERVICE_COLS="all_collection_26-09-2022 service_26-09-2022"

```
Always there is an option to add defaults and env.template. What is your opinon?

Those **input** directories like`DATASET_DIR_PATH`, `PUBLICATION_DIR_PATH`, etc. can not be placed inside `OUPUT_DIR_PATH` - this is the **output** directory which is cleared entirely in every iteration of the program. 
But what your comment remind me of is changing names for:
`TRAINING_FILE_PATH` -> `TRAINING_DIR_PATH`
`SERVICE_FILE_PATH` -> `SERVICE_DIR_PATH`
In that case I will just remove "DIR" part from all variables just to make it shorter

Reply:
In the next commit I will add:
1) defaults for input data
2) env template
3) make env path variables shorter